x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -354,15 +354,14 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
       let signal: { name: string | null } = { name: null };
       try {
-        // TODO: Once we are past experimental phase this code should be removed
-        // TODO: This currently prevents TGrid from refreshing
-        if (this.experimentalFeatures.ruleRegistryEnabled) {
-          signal = { name: DEFAULT_ALERTS_INDEX };
-        } else {
-          signal = await coreStart.http.fetch(DETECTION_ENGINE_INDEX_URL, {
+        const { index_name: indexName } = await coreStart.http.fetch(
+          `${BASE_RAC_ALERTS_API_PATH}/index`,
+          {
             method: 'GET',
-          });
-        }
+            query: { features: SERVER_APP_ID },
+          }
+        );
+        signal = { name: indexName[0] };
```

Comment:
nit - let's add a comment for FUTURE DEVELOPER, that the expectation here is to only have one index for the alert index

Reply:
Def! Will add.

Reply:
I didn't do...BUT will add if CI fails or as follow up.